buildh/lipids/Berger_POPC.json

```diff
@@ -0,0 +1,43 @@
+{
+  "resname": ["POPC", "PLA", "POP"],
```

Comment:
Maybe we should have 3 separate files instead? (`Berger_POPC.json`, `Berger_PLA.json` and `Berger_POP.json`)

Reply:
Files will be redundant but this will more clear I guess.

Reply:
But if we need to update the atoms data, we will need to modify each files.
I agree this solution is more clear when reading the filename but it could be stated in the documentation.

Tough choice :)

Reply:
In my opinion, I'd only keep `Berger_POPC.json`, that is, the force field name followed by the common lipid name. Then, the residue name in the pdb or gro file is something else that can be easily handled with  `"resname": ["POPC", "PLA", "POP"],`. Sometimes, you can have more than one lipids that are called POP, e.g. POPC and POPE (I've already seen that even if it's bad). That's because in GROMACS what matters is the order of the atoms and their declaration in the top file. Thus 2 different residues can be called POP! If we use this trick to pass a list to the `resname` key, we tackle this problem. Moreover, it'll avoid having too many .json files.

Reply:
Okay good, I leave it as is.